src/alert.cpp

```diff
@@ -1965,21 +1966,16 @@ namespace libtorrent
 	std::vector<tcp::endpoint> dht_get_peers_reply_alert::peers() const
```

Comment:
thinking again, I believe `tcp::endpoint` is right, since this is endpoint the peer is announced (per hash) in the DHT